pgrx-pg-sys/include/pg12.h

```diff
@@ -86,6 +87,8 @@
 #include "nodes/makefuncs.h"
 #include "nodes/nodeFuncs.h"
```

Comment:
To avoid relying on transitive inclusion, for my purposes:
```suggestion
#include "nodes/miscnodes.h"
#include "nodes/nodeFuncs.h"
```